### PR TITLE
SOC-2113 | Discusssions - fix handling empty forums

### PR DESCRIPTION
**Pull request by kvas-damian**
## Links

* https://wikia-inc.atlassian.net/browse/SOC-2113

## Description

Fix handling empty forums

## Reviewers

@bkoval @rafalkalinski

### CODE REVIEW

**Comment by armonr** [comment]
Thanks @kvas-damian

File: front/main/app/models/discussion-forum.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ DiscussionForumModel.reopenClass({
 			url: M.getDiscussionServiceUrl(`/${wikiId}/forums/${forumId}`),
 			success: (data) => {
 				const contributors = [],
-					posts = data._embedded ? data._embedded['doc:threads'] : [],
+					posts = (data._embedded && data._embedded['doc:threads']) ? data._embedded['doc:threads'] : [],
```
Comment (bkoval) [inline front/main/app/models/discussion-forum.js]: https://github.com/Wikia/mercury/commit/8bee903300a03c6b57d5231f4c65af47dcd2bb48 We already have a fix for empty forums here and it looks similar.

Comment (kvas-damian) [inline front/main/app/models/discussion-forum.js]: ;)

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by kvas-damian** [comment]
Fix was already merge to dev: https://github.com/Wikia/mercury/pull/2132#discussion_r55386344

**Pull request closed without merging**